File: textToSpeeech/test_text_to_speech.js
const textToSpeech = require('@google-cloud/text-to-speech');

const fs = require('fs');
const util = require('util');
main()
async function main() {
  console.log('Start encoding\n');
  const client = new textToSpeech.TextToSpeechClient();

  const text = 'Bonjour, ceci est un example de conversion.';
  const request = {
    input: {text: text},
    voice: {languageCode: 'fr-FR', ssmlGender: 'NEUTRAL'},
    audioConfig: {audioEncoding: 'MP3'},
  };
  const [response] = await client.synthesizeSpeech(request);
  const writeFile = util.promisify(fs.writeFile);
  await writeFile('output.mp3', response.audioContent, 'binary');
  console.log('Audio content written to file: output.mp3');
}

File: textToSpeeech/test_text_to_speech.cs
using System;
using System.IO;
using Google.Cloud.TextToSpeech.V1;

public class QuickStart
{
    public static void Main(string[] args)
    {
        TextToSpeechClient client = TextToSpeechClient.Create();
        SynthesisInput input = new SynthesisInput
        {
            Text = "Dorian est la plus belle personne du languedoc roussillon!"
        };
        VoiceSelectionParams voice = new VoiceSelectionParams
        {
            LanguageCode = "fr-FR",
            SsmlGender = SsmlVoiceGender.Neutral
        };
        AudioConfig config = new AudioConfig
        {
            AudioEncoding = AudioEncoding.Mp3
        };
        var response = client.SynthesizeSpeech(new SynthesizeSpeechRequest
        {
            Input = input,
            Voice = voice,
            AudioConfig = config
        });
        using (Stream output = File.Create("output.mp3"))
        {
            response.AudioContent.WriteTo(output);
            Console.WriteLine($"Audio content written to file 'output.mp3'");
        }
    }
}